.circleci/integration-tests/Dockerfile

```diff
@@ -28,12 +38,28 @@ RUN curl -o kubectl https://s3.us-west-2.amazonaws.com/amazon-eks/1.22.6/2022-03
     && chmod +x ./kubectl \
     && mv ./kubectl /usr/local/bin
 
+#install Apache Hive 2.3.9 (Download same version of library to match EMR hive version created by DAG)
+RUN curl "https://downloads.apache.org/hive/hive-2.3.9/apache-hive-2.3.9-bin.tar.gz" -o  "/tmp/apache-hive-2.3.9-bin.tar.gz"  \
+    && tar -xf /tmp/apache-hive-2.3.9-bin.tar.gz -C /usr/local
+ENV HIVE_HOME=/usr/local/apache-hive-2.3.9-bin
+
+#install Apache Hadoop 2.10.1 (Download same version of library to match EMR hadopp version created by DAG)
+RUN curl "https://archive.apache.org/dist/hadoop/common/hadoop-2.10.1/hadoop-2.10.1.tar.gz" -o "/tmp/hadoop-2.10.1.tar.gz" \
+    && tar -xf /tmp/hadoop-2.10.1.tar.gz -C /usr/local
+ENV HADOOP_HOME=/usr/local/hadoop-2.10.1
+ENV HADOOP_CONF_DIR=$HADOOP_HOME/conf
+
+RUN curl "https://raw.githubusercontent.com/apache/hive/master/data/conf/hive-site.xml" -o "/tmp/hive-site.xml" \
```

Comment:
why do we need this, if we are not using EMR version of hive-site.xml?

Reply:
removed not needed. It worked with out it